game-core/src/main/java/games/strategy/triplea/attachments/UnitSupportAttachment.java

```diff
@@ -341,22 +333,14 @@ static void addRule(final UnitType type, final GameData data, final boolean firs
     final String attachmentName =
         (first ? Constants.SUPPORT_RULE_NAME_OLD_TEMP_FIRST : Constants.SUPPORT_RULE_NAME_OLD) + type.getName();
     final UnitSupportAttachment rule = new UnitSupportAttachment(attachmentName, type, data);
-    rule.setBonus("1");
+    rule.setBonus(1);
     rule.setBonusType(Constants.OLD_ART_RULE_NAME);
     rule.setDice("strength");
     rule.setFaction("allied");
-    rule.setImpArtTech("true");
-    if (first) {
-      rule.setNumber("0");
-    } else {
-      rule.setNumber("1");
-    }
+    rule.setImpArtTech(String.valueOf(true));
```

Comment:
If you're changing this line anyway, could you not just call `setImpArtTech(Boolean)` directly and let auto-boxing kick in?

```java
rule.setImpArtTech(true);
```